main_test.go

```diff
@@ -35,39 +183,572 @@ func TestExample2(t *testing.T) {
+	//1   |laptop                    |0             |                          |
+}
+
+// 3. Command Syntax: `store create-order <customer_id> <product_id>`
+
+// This command should add a new order to the orders table.
+// The customer_id and product_id arguments are required.
+
+// Example:
+
+//	> store create-order 1 3
+//	ORDER_ID      CUSTOMER_ID         PRODUCT_ID
+//	1             1                   3
+//
+// 1. creating new order with all inputs valid
+// 2. creating new order missing customer id
+// 3. creating new order missing product id
+// 4. checking correct print status
+// 5. creating new order with foreign key violation in customer-id
+// 6. creating new order with foreign key violation in product-id
+// 7. creating new order with customer_id not an integer
+// 8. creating new order with product_id not an integer
+func TestCreateNewOrderMissingProductId(t *testing.T) {
+
+	err := createOrdersData([][]string{}, [][]string{}, [][]string{{"store", "create-order", "1"}})
+	assert.ErrorContains(t, err, "Must specify product_id and customer_id")
+}
+
+func TestCreateNewOrderMissingCustomerId(t *testing.T) {
+	err := createOrdersData([][]string{}, [][]string{}, [][]string{{"store", "create-order", "12"}})
+	assert.ErrorContains(t, err, "Must specify product_id and customer_id")
+}
+
+func TestCreateNewOrderCustomerIdIsFloat(t *testing.T) {
+
+	err := createOrdersData([][]string{}, [][]string{}, [][]string{{"store", "create-order", "1", "4.5"}})
+	assert.ErrorContains(t, err, "Must be valid integer")
+}
+
+func TestCreateNewOrderProductIdIsFloat(t *testing.T) {
+	err := createOrdersData([][]string{}, [][]string{}, [][]string{{"store", "create-order", "1.5", "4"}})
+	assert.ErrorContains(t, err, "Must be valid integer")
+}
+
+func TestCreateNewOrderWithValidInputs(t *testing.T) {
+	err := tearDownOrders()
+	assert.NilError(t, err)
+
+	db, err := connectDB()
+	assert.NilError(t, err)
+
+	err = createOrdersData([][]string{{"store", "create-customer", "vivek.s@outreach.io", "WA"}}, [][]string{{"store", "create-product", "--sku=abcde", "banana", "5"}}, [][]string{{"store", "create-order", "1", "1"}})
+
+	var orderID int
+	var createdAt sql.NullString
+	var customerID int
+	var productID int
+	var sku sql.NullString
+
+	rows := db.QueryRow("SELECT * FROM Orders ORDER BY ID DESC LIMIT 1")
+
+	assert.NilError(t, rows.Err())
+
+	err = rows.Scan(&orderID, &createdAt, &customerID, &productID, &sku)
+	assert.NilError(t, err)
+	assert.Equal(t, productID, 1)
+	assert.Equal(t, customerID, 1)
+	assert.Equal(t, orderID, 1)
+}
+
+func TestCreateNewOrder_ValidInputs_CustomerForeignKey_Error(t *testing.T) {
+	err := tearDownOrders()
+	assert.NilError(t, err)
+
+	err = createOrdersData([][]string{}, [][]string{{"store", "create-product", "--sku=abcde", "banana", "5"}}, [][]string{{"store", "create-order", "1", "1"}})
+
+	assert.ErrorContains(t, err, "Violates Foreign Key constraints")
+}
```

Comment:
From a UX perspective, perhaps we should try and convert this error into something more user friendly? For example, we could have error messages like:
```
Error: invalid product id: product with id "2" does not exist
...
Error: invalid customer id: customer with id "2" does not exist
```